src/KNXClient.ts

```diff
@@ -87,20 +85,16 @@ export type KNXClientOptions = {
 	ipAddr?: string
 	/** The port, default is "3671" */
 	ipPort?: number | string
-	/** Default: "TunnelUDP". "Multicast" if you're connecting to a KNX Router. "TunnelUDP" for KNX Interfaces, or "TunnelTCP" for secure KNX Interfaces (not yet implemented) */
+	/** Default: "TunnelUDP". "Multicast" if you're connecting to a KNX Router. "TunnelUDP" for KNX Interfaces */
 	hostProtocol?: KNXClientProtocol
-	/** True: Enables the secure connection. Leave false until KNX-Secure has been released. */
-	isSecureKNXEnabled?: boolean
 	/** Avoid sending/receive the ACK telegram. Leave false. If you encounter issues with old interface, set it to true */
 	suppress_ack_ldatareq?: boolean
 	/** Leave true forever. This is used only in Node-Red KNX-Ultimate node */
-	localEchoInTunneling?: boolean
+	localEchoInTunnelling?: boolean
```

Comment:
remember to mark this PR as a breaking change as you changed the name of a property here. List this change in PR body